src/geometry/PolySetUtils.cc

```diff
@@ -129,4 +141,38 @@ bool is_approximately_convex(const PolySet& ps) {
 #endif
 }
 
+
+PolySet  convert_polyset(const shared_ptr<const Geometry>& geom)
+{
+  PolySet ps(3);
+  if (const auto geomlist = dynamic_pointer_cast<const GeometryList>(geom)) {
+    for (const Geometry::GeometryItem& item : geomlist->getChildren()) {
+      return convert_polyset(item.second);
```

Comment:
Not sure I understand the logic: You just return the polygons from the first geometry in the list?

Reply:
yep, this is the error which made my ctest fail